Cargo.toml

```diff
@@ -187,3 +187,9 @@ blocks_in_conditions = "allow"
 [[bin]]
 name = "spin"
 path = "src/bin/spin.rs"
+
+# Remove this once Wasmtime 35.0.0 is released
```

Comment:
Given that this is only needed for `te` header support, could we separate this part out into a different PR?